src/workers/cdc.ts

```diff
@@ -368,6 +376,13 @@ const onPostChange = async (
       }
     }
 
+    if (data.payload.after.type === PostType.Collection) {
+      const collection = data as ChangeMessage<CollectionPost>;
+      if (isCollectionUpdated(collection)) {
+        await notifyPostCollectionUpdated(logger, data.payload.after);
```

Comment:
Reminder to create the topic in the streams project

Reply:
It exists. 👍